### ISSUE TITLE
Order by storage filename bugs Project frontend and persists state until manual change

### ISSUE THREAD

**Issue by Gornoka** [heartexlabs/label-studio#4025]
**Describe the bug**
When trying to order by storage filename the front end returns an error message instead of ordering and stores the broken order.

( only tested with s3 image storage)

**To Reproduce**
1. Create project with s3 cloud storage (only tested with local minio)
2. Upload and sync some images
3. Go to project overview
4. Order by storage filename
5. See error
6. When opening the project again it now errors again and does not load any tasks (see video) 

https://user-images.githubusercontent.com/45569834/231716199-31f2b67e-4817-4ae8-bddf-d22f3f2d9261.mp4

**Expected behavior**
If ordering by a selected key is not possible it should lead to a normal error message and reverting to a valid save, not in the UI getting borked into some weird state

**Screenshots**
https://user-images.githubusercontent.com/45569834/231716199-31f2b67e-4817-4ae8-bddf-d22f3f2d9261.mp4
![image](https://user-images.githubusercontent.com/45569834/231716546-8495fd64-1ee3-42c3-959b-d8516e967136.png)

**Environment (please complete the following information):**
 - OS: Windows
 - Label Studio Version [e.g. 1.7.2]
 - Running in Docker
**Additional context**
Traceback:
````logs
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/django/utils/decorators.py", line 43, in _wrapper
    return bound_method(*args, **kwargs)
  File "/label-studio/label_studio/./data_manager/api.py", line 207, in get
    page = self.paginate_queryset(queryset)
  File "/usr/local/lib/python3.10/dist-packages/rest_framework/generics.py", line 171, in paginate_queryset
    return self.paginator.paginate_queryset(queryset, self.request, view=self)
  File "/label-studio/label_studio/./data_manager/api.py", line 136, in paginate_queryset
    return super().paginate_queryset(queryset, request, view)
  File "/usr/local/lib/python3.10/dist-packages/rest_framework/pagination.py", line 216, in paginate_queryset
    return list(self.page)
  File "/usr/local/lib/python3.10/dist-packages/django/core/paginator.py", line 177, in __len__
    return len(self.object_list)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/query.py", line 262, in __len__
    self._fetch_all()
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/query.py", line 1324, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/query.py", line 51, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/compiler.py", line 1162, in execute_sql
    sql, params = self.as_sql()
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/compiler.py", line 513, in as_sql
    extra_select, order_by, group_by = self.pre_sql_setup()
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/compiler.py", line 56, in pre_sql_setup
    order_by = self.get_order_by()
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/compiler.py", line 372, in get_order_by
    resolved = expr.resolve_expression(self.query, allow_joins=True, reuse=None)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/expressions.py", line 247, in resolve_expression
    c.set_source_expressions([
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/expressions.py", line 248, in <listcomp>
    expr.resolve_expression(query, allow_joins, reuse, summarize)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/expressions.py", line 578, in resolve_expression
    return query.resolve_ref(self.name, allow_joins, reuse, summarize)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1779, in resolve_ref
    join_info = self.setup_joins(field_list, self.get_meta(), self.get_initial_alias(), can_reuse=reuse)
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1648, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
  File "/usr/local/lib/python3.10/dist-packages/django/db/models/sql/query.py", line 1562, in names_to_path
    raise FieldError("Cannot resolve keyword '%s' into field. "
django.core.exceptions.FieldError: Cannot resolve keyword 'storage_filename' into field. Choices are: annotations, cancelled_annotations, comment_authors, comment_count, created_at, data, drafts, file_upload, file_upload_id, id, inner_id, io_storages_azureblobimportstoragelink, io_storages_gcsimportstoragelink, io_storages_localfilesimportstoragelink, io_storages_redisimportstoragelink, io_storages_s3importstoragelink, is_labeled, last_comment_updated_at, locks, meta, overlap, predictions, project, project_id, total_annotations, total_predictions, unresolved_comment_count, updated_at, updated_by, updated_by_id
````

**Comment by makseq** [comment]
Thank you for reaching us, it's a well known bug, we have a ticket about it ([LSDV-4876](https://labelstudio.aha.io/develop/features/LSDV-4876). We will let you know, when this is fixed.

**Comment by klinsc** [comment]
Same here

**Comment by jombooth** [comment]
Hi @Gornoka and @klinsc - this was fixed in our latest release, [1.11.0](https://github.com/HumanSignal/label-studio/releases/tag/1.11.0)!

**Comment by Gornoka** [comment]
thanks for the update.

> Hi @Gornoka and @klinsc - this was fixed in our latest release, [1.11.0](https://github.com/HumanSignal/label-studio/releases/tag/1.11.0)!
